services/src/api/model/db_types.rs

```diff
@@ -0,0 +1,408 @@
+use ordered_float::NotNan;
+use postgres_types::{FromSql, ToSql};
+
+use crate::{
+    error::Error,
+    projects::{
+        ColorParam, DerivedColor, DerivedNumber, LineSymbology, NumberParam, PointSymbology,
+        PolygonSymbology, RasterSymbology, Symbology,
+    },
+};
+
+use super::datatypes::{
+    Breakpoint, Colorizer, DefaultColors, LinearGradient, LogarithmicGradient, OverUnderColors,
+    Palette, RgbaColor,
+};
+
+#[derive(Debug, ToSql, FromSql)]
+#[postgres(name = "DefaultColors")]
+pub struct DefaultColorsDbType {
```

Comment:
does it make sense to have "Type" in the name? Isn't it obvious that this is a type?